src/briefcase/platforms/linux/appimage.py

```diff
@@ -33,6 +34,9 @@ def appdir_path(self, app):
     def project_path(self, app):
         return self.bundle_path(app)
 
+    def local_requirements_path(self, app):
+        return self.bundle_path(app) / "_local"
```

Comment:
`_requirements` would be a more descriptive name.

Reply:
Yeah - I went round in circles a few times on this name. It's a bike shed; `_requirements` works for me.